Remove future-facing statement from import page (#31686)
hashicorp/terraform: website/docs/cli/import/usage.mdx

> The `terraform import` command is used to import existing infrastructure.

Author: ```suggestion
Use the `terraform import` command to import existing infrastructure to Terraform state.
```

The `terraform import` command is used to import existing infrastructure.

> The command can only import one resource at a time. This means
> you cannnot import an entire collection of resources such as an AWS VPC at one time. 

Author: ```suggestion
The command can only import one resource at a time. This means you cannot simultaneously import an entire collection of resources, like an AWS VPC.
```
